### DISCUSSION TITLE
Deploy failed: GitHub Pages

### DISCUSSION THREAD

**Discussion by romw314** [community/community#60494] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have deploy failed in my romw314/chess-no-25 repo to GitHub Pages.

**Comment by yoannchaudet** [comment]
Your workflow run has an annotation with the error: https://github.com/romw314/chess-no-25/actions/runs/5507533987.

Looks like invalid YAML to me (missing indentation).

**Reply by romw314** [reply]
And in the description of the `actions/upload-pages-artifact` action, the artifact must be a tgz file and in older commits, my workflow was used `actions/upload-artifact` for a directory: https://github.com/romw314/chess-no-25/blob/08f7d6f425f3c456b6b2075aaaab36d838dd0d18/.github/workflows/react-deploy.yml
